gradle/libs.versions.toml

```diff
@@ -16,7 +16,7 @@ junit = "4.13.2"
 mockk = "1.13.9"
 appcompat = "1.7.0"
 material = "1.12.0"
-ucrop = "2.2.9"
+ucrop = "2.2.9-native"
```

Comment:
I had to switch to the native version because the compression wouldn't work for me and the uploaded avatar would be the same size as the original image despite the compression.

Reply:
From uCrop's documentation: 
> implementation 'com.github.yalantis:ucrop:2.2.8-native' - get power of the native code to preserve image quality (+ about 1.5 MB to an apk size)

1.5 MB is a lot. Is there any workaround?

Reply:
I haven't found one in this library. We can handle the compression ourselves on top of it though, but that will be tricky as we have to be careful not to duplicate the compression when the one from the library works (I imagine this might be dependent on the system version for example). 

Using the `-native` variant seemed like a better pick for now to not spend too much time on it.

Reply:
@maxme I'm curious to know what you think of this

Reply:
Adding 1.5Mb is not ideal, but what worries me most is adding native code to our library dependencies. For deployments on the app store, it's going to be extra steps, and might be limiting for our users. They might be surprised they have native code and might look for the culprit dependency.

Reply:
> I had to switch to the native version because the compression wouldn't work for me and the uploaded avatar would be the same size as the original image despite the compression.

Do you get some logs or is this a silent error (they skip the compression steps maybe)?

Reply:
I haven't seen any errors (I haven't looked for them though at that point) just that the final image had the exact same size as the photo taken with the camera. I will take another look at it.

Reply:
I'm testing this on different devices/emulators and the image size is always the same. The compression seems to work in a way that you can clearly see the drop in the image quality and the width/height sizes. Yet the file size remains unchanged.